vignettes/adpet.Rmd

```diff
@@ -0,0 +1,374 @@
+---
+title: "Creating ADTPET and ADAPET"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Creating ADTPET and ADAPET}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(admiraldev)
+```
+
+# Introduction
+
+This article describes the creation of PET Scan ADaMs.
+
+We recommend splitting out Amyloid and Tau into two distinct ADaMs datasets. The vignette presents
```

Comment:
We use lower case "amyloid" and "tau" in sentences, unless they are at the beginning of a sentence.